examples/golang/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+services:
+
+  proxy:
+    build:
+      context: .
+      dockerfile: Dockerfile-proxy
+    depends_on:
```

Comment:
probably better to make this `depends_on` `service_healthy`

Reply:
good

Reply:
Done.